### PR TITLE
add api endpoints and models for definition page

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony Please review

**Comment by g1039** [comment]
@miltontony Ready for review

**Comment by miltontony** [comment]
👍 🍰

File: dashboards/models.py [hunk 1]
```diff
@@ -119,3 +119,26 @@ class Meta:
...
+    created_by = models.ForeignKey(User, related_name='definition_created',
+                                   null=True, blank=True)
+    updated_by = models.ForeignKey(User, related_name='definition_updated',
+                                   null=True, blank=True)
+    user = property(lambda self: self.created_by)
+
+    def __str__(self):  # __unicode__ on Python 2
+        return "%s" % self.title
+
+    class Meta:
```
Comment (miltontony) [inline dashboards/models.py]: You don't need to override `verbose_name` and `verbose_name_plural`

**Comment by miltontony** [comment]
👍

**Comment by g1039** [comment]
@miltontony Updated. . . . Please have a look.

**Pull request merged**